examples/proc_fieldSweep.py

```diff
@@ -1,142 +1,29 @@
-    # relative offsets and the frequency shifts needed to align the signals, at
-    # the end.
-    all_offsets = zeros(len(s.getaxis("indirect")))
-    for z in range(len(s.getaxis("indirect"))):
-        fl.plot(s["indirect", z], ax=ax_list[1])
-        if z == 0:
-            ax_list[1].axvline(color="k", x=freq_slice[0])
-            ax_list[1].axvline(color="k", x=freq_slice[-1])
-        offset = s["indirect", z].C.mean("nScans").argmax("t2").item()
-        all_offsets[z] = offset
-        carrier_freq_MHz = s.getaxis("indirect")[z]["carrierFreq"]
-        field = s.getaxis("indirect")[z]["Field"]
-        nu_rf = carrier_freq_MHz - offset / 1e6  # because in order to make my
-        #                                       offset more negative, I
-        #                                       increase my field, or decrease
-        #                                       my carrier -- in other words
-        #                                       the frequency axis for our FID
-        #                                       is the negative of (ν_RF-ν_carrier)
-        nu_NMR.append(nu_rf)
-    ax_list[1].set_title("Field Slicing")
-    s.ift("t2")
-    s *= exp(
-        -1j
-        * 2
-        * pi
-        * nddata(all_offsets, [-1], ["indirect"])
-        * s.fromaxis("t2")
-    )
-    s.ft("t2")
-    # }}}
-    fl.plot(s.C.mean("nScans"), ax=ax_list[2])
-    s = s["t2":freq_slice].mean("nScans").integrate("t2")
-    # }}}
-    # {{{convert x axis to ppt = v_NMR/v_ESR
-    ppt = nu_NMR / nu_B12_GHz
-    s.setaxis("indirect", ppt)
-    s.set_units("indirect", "MHz/GHz")
-    s.rename("indirect", "resonance ratio")
-    # }}}
-    # {{{Fitting
-    fl.plot(s, "o-", ax=ax_list[3])
-    ax_list[3].set_title("Field Sweep ppt")
-    fitting = s.polyfit("resonance ratio", order=2)
-    field_fine = nddata(
-        r_[
-            s.getaxis("resonance ratio")[0] : s.getaxis("resonance ratio")[
-                -1
-            ] : 100j
-        ],
-        "resonance ratio",
-    )
-    polyline = fl.plot(
-        field_fine.eval_poly(fitting, "resonance ratio"),
-        label="fit",
-        ax=ax_list[3],
+        postproc=post_proc,
+        lookup=prscr.lookup_table,
     )
-    fig.tight_layout(rect=[0, 0.03, 1, 0.95])
-    _, B, A = fitting
-    center = -B / 2 / A
-    # }}}
-    logger.info(strm("ESR frequency is %f" % (nu_B12_GHz)))
-    logger.info(strm("MHz/GHz value at peak/dip appears to be:", center))
-    ax_list[3].axvline(x=center, color=polyline[0].get_color(), ls=":")
-fl.show()
+    use_freq = False
+    if use_freq:
+        s["indirect"] = s["indirect"]["carrierFreq"] * 1e6
+        s.set_units("indirect", "Hz")
+    else:
+        # I wanted to use the carrier (use_frep=True), but it seems like the
+        # first point isn't stored properly
+        s["indirect"] = s["indirect"]["Field"]
+        s.set_units("indirect", "G")
+    prscr.rough_table_of_integrals(s, fl=fl)
```

Comment:
note that I'm making comments about stuff that gets removed currently before this, but this block would actually come before that.